src/errors/checkXcmTxInputs.ts

```diff
@@ -666,11 +658,12 @@ export const checkParaAssets = async (
 	} else {
 		// not a valid number
 		// check if id is a valid token symbol of the system parachain chain
-		if (!invalidNumber) {
+		// TODO: Im confused about this, clarify this logic.
```

Comment:
@marshacb Lets go over this in a call because I am not sure I understand the conditional below.

Will that if statement ever be hit? I think we might have talked about this in the past but it would be nice to have a refresher.

Reply:
If you look at the diff to the left, if it doesnt enter that first `if` it will never hit the second one inside of the `else` statement. Or atleast that is what I understand here.

Reply:
We figured out what we need to do in a conversation and a PR will be up shortly with the fixes